src/js/lib/jquery.kazoosdk.js

```diff
@@ -346,6 +346,10 @@
 			qubicleReports: {
 				'get': { verb: 'POST', url: 'accounts/{accountId}/qubicle_reports' }
 			},
+			qubicleDashboards: {
```

Comment:
This new object should look like this to follow the existing conventions:
```javascript
			dashboards: {
				'getQueueDetails': { verb: 'GET', url: 'accounts/{accountId}/dashboards/queue_details/{queueId}' },
				'getQueueOverview': { verb: 'GET', url: 'accounts/{accountId}/dashboards/queue_overview' }
			},
```
This convention allows us to "guess" the resource's format without having to look at the sdk.

Also, please try to keep that list alphabetically sorted and move that object to the appropriate spot once renamed.